scripts/setup-fix.bat

```diff
@@ -0,0 +1,6 @@
+(	python --version>nul 2>&1 && (
+	echo Python environment found.
```

Comment:
just out of curiosity: was the double quote change @yliaog  suggested reverted for some reason?

Reply:
> 
> 
> just out of curiosity: was the double quote change @yliaog suggested reverted for some reason?

works fine without double qoute, so thought of reverting the change :)